docs/ALLOWED_HOSTS_AND_COMMANDS.md

```diff
@@ -1,36 +1,65 @@
-| Language / Platform                   | Project Types                                                                  | Allowed Commands | Remote Hosts Typically Accessed                 |
+# List of allowed hosts and commands
+
+To allow commands and hosts correctly, make sure to add Operating System specific [commands](#Platform-Specific Commands) along with common [commands](#Common Commands (All Platforms)) if there is any!
+
+## Common Commands (All Platforms)
+
+| Language / Platform | Project Types | Allowed Commands | Remote Hosts Typically Accessed |
```

Comment:
How about `External Commands` instead of `Allowed Commands`. `Remote Hosts` could be the last column.